playbooks/robusta_playbooks/node_enrichments.py

```diff
@@ -121,6 +123,7 @@ def node_health_watcher(event: NodeChangeEvent):
         source=FindingSource.KUBERNETES_API_SERVER,
         aggregation_key="node_not_ready",
         severity=FindingSeverity.MEDIUM,
+        subject=KubeObjFindingSubject(event.obj)
```

Comment:
Maybe it's just better to have NodeFindingSubject

Reply:
NodeFindingSubject is too specific and would technically only be used in one place, 
KubeObjFindingSubject also gets the kind and metadata from the object to make the finding subject so its more diverse

Reply:
ok